### PR TITLE
Verify email address is verified before allowing payment method to be available

### LINKED ISSUE #9: Ensure the email is verified before activating the payment method
_Related: https://github.com/nebulab/solidus_paypal_commerce_platform/issues/7_

This can take the form of a button in the payment-method page that will trigger a new check on the verification, much like what happens when configuring a DNS and you're waiting for the nameservers to be updated.

The mockup is not binding in any way, but just a way to get started

image

### PR DESCRIPTION
**Pull request by seand7565**
We want to ensure that the users PayPal email is verified before
accepting payments with this payment_method, so we're defaulting
payment method availability to false and requiring the user to verify
that their email is confirmed before allowing the payment_method
to be available to users.

image

Fixes #9

### CODE REVIEW

File: app/views/solidus_paypal_commerce_platform/admin/payment_methods/_available_to_users.html.erb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<% if @payment_method.type == "SolidusPaypalCommercePlatform::PaymentMethod" && !@payment_method.preferences[:paypal_email_confirmed] %>
+  <%= f.check_box :available_to_users, disabled: true %>
+  <span class="hint-tooltip" title="" data-content="You must verify that your email is confirmed with PayPal before allowing users to check out.">
+    <i class="fa fa-info-circle"></i>
+  </span>
```
Comment (elia) [inline app/views/solidus_paypal_commerce_platform/admin/payment_methods/_available_to_users.html.erb:5]: Would be nice to fully support translations with

```suggestion
  <%= f.field_hint :available_to_users %>
```
plus adding `spree.hints.solidus_paypal_commerce_platform/payment_method.available_to_users` to `locales.yml`

Comment (seand7565) [reply app/views/solidus_paypal_commerce_platform/admin/payment_methods/_available_to_users.html.erb:5]: Of course, I always forget the translations. Also, I had no idea `f.field_hint` was a thing, that's seriously cool. Thanks!

**Comment by seand7565** [comment]
Not sure if GitHub is having issues or if the problem is on my end, but pushing to my branch didn't update this PR. Closing this one to get an updated PR up!

**Comment by seand7565** [comment]
Oookay, reopening - as soon as I closed it the PR updated. 🤔

**Pull request merged**